client/cli/lib/init.ts

```diff
@@ -9,7 +8,7 @@ const dirname = process
   .pop() as string;
 
 export default () =>
-  execa('npm', ['run', 'plop', 'component', process.cwd(), uppercamelcase(dirname)], {
+  execa('npm', ['run', 'plop', 'component', process.cwd(), dirname], {
```

Comment:
Should we do lower tho?

Reply:
not sure... at this point it just shows the name of the folder... I'm running lower in the name property in package.json file... I'm considering whether or not we should auto kebab-case it...?

Reply:
Nah I think it's fine to leave it be